GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -133,52 +121,121 @@ private static void UpdateGuiFocus(ToolBarSelectedButton currentTab)
 
         private void OnGuiCreateBundleSelect()
         {
-            EditorGUILayout.LabelField("AssetBundle Creation", EditorStyles.boldLabel);
+            var descriptionTextStyle = new GUIStyle(GUI.skin.label)
+            {
+                fontStyle = FontStyle.Italic,
+                wordWrap = true
+            };
+            
+            EditorGUILayout.LabelField("Create AssetBundle", EditorStyles.boldLabel);
+            
+            EditorGUILayout.BeginVertical("textfield");
```

Comment:
Create a const field for "TextField" to use in each case here and below. Give it a // style comment explaining what it does to the style and provide a link here: https://docs.unity3d.com/ScriptReference/GUILayout.TextField.html

Reply:
This would be a more correct documentation link: https://docs.unity3d.com/ScriptReference/GUISkin-textField.html

But thanks, resolved!